misc/python/materialize/cli/ci_closed_issues_detect.py

```diff
@@ -61,15 +65,28 @@
     | \(\#1\)\ IS\ NULL
     # test/sqllogictest/cockroach/*.slt
     | cockroach\#
+    | Liquibase
     # cloud repo
     | cloud\#
     # ci/test/lint-buf/README.md
     | Ignore\ because\ of\ #99999
+    # src/persist-client/src/internal/machine.rs
+    | discussion\ of\ this\ in
+    # src/transform/tests/testdata/partial-reduction-pushdown
+    | issues/10119,
+    # src/transform/src/normalize_lets.rs
+    | issues/21145$
+    # src/storage-client/src/controller.rs
+    | issues/20211\>
+    # src/sql/src/plan/statement.rs
+    | issues/20019\>
```

Comment:
<s>Not sure if it is a good idea to reference individual issues here. Why not add a TODO there?</s>

Oh, I see, these are ignores.

Reply:
What about adding "See " as ignore pattern and use it in these locations?

Like in:
```
                // See <https://github.com/MaterializeInc/materialize/issues/20211>.
```

Reply:
Done for the 2 locations that will stay. For the 2 that I asked devs to fix left the ignore in for now.